modules/monitoring-default-monitoring-components.adoc

```diff
@@ -11,40 +11,51 @@ By default, the {product-title} {product-version} monitoring stack includes thes
 [options="header"]
 |===
 
-|Component|Description
+|Component|Description|Version
 
 |Cluster Monitoring Operator
 |The Cluster Monitoring Operator (CMO) is a central component of the monitoring stack. It deploys and manages Prometheus instances, the Thanos Querier, the Telemeter Client, and metrics targets and ensures that they are up to date. The CMO is deployed by the Cluster Version Operator (CVO).
+|v0.1.1
 
 |Prometheus Operator
 |The Prometheus Operator (PO) in the `openshift-monitoring` project creates, configures, and manages platform Prometheus instances and Alertmanager instances. It also automatically generates monitoring target configurations based on Kubernetes label queries.
+|v0.40.0
 
 |Prometheus
 |Prometheus is the monitoring system on which the {product-title} monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
+|v2.20.0
 
 |Prometheus Adapter
 |The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal pod autoscaling. The Prometheus Adapter is also used by the `oc adm top nodes` and `oc adm top pods` commands.
+|v0.4.1
```

Comment:
not sure about the version, need confirm

Reply:
it might be v0.7.0 but @s-urbaniak should confirm

Reply:
@s-urbaniak can you please confirm the Prometheus Adapter version for 4.6? Thank you!

Reply:
Correct, it is 0.7.0